src/me/StevenLawson/TotalFreedomMod/Listener/TFM_BlockListener.java

```diff
@@ -111,6 +112,14 @@ else if (playerLocation.distanceSquared(location) > (nukeMonitorRange * nukeMoni
                 }
             }
         }
+        if (player.getGameMode().equals(GameMode.SURVIVAL))
+        {
+            for (ItemStack stack : event.getBlock().getDrops())
+            {
+                event.getBlock();
+                player.getInventory().addItem(stack);
```

Comment:
This does not take into account if entity wiper is enabled or not. If it isn't, this might result in duplicate drops.